src/docs/Foundations/MagicUnit/code.mdx

```diff
@@ -137,3 +137,13 @@ It basically just do the `to-rem()` part for you
 ### The shorthands variables :
 
 > the **shorthands variables** behave exactly as the magic-unit-rem, but in a variable form factor.
+
+### The magic-unit grid preview :
+
+In order to help you to position your elements as well as possible according to the magic-unit, Mozaic provides the css class `mu-preview-grid` to add on your container and which allows the display of a positioning grid.
```

Comment:
update docs

Reply:
"The code of this class will not be compiled into the production build when the `MOZAIC_ENV=production` environment variable is provided. "

I think it needs to be added before the preview. People may not read after it.

Reply:
fixed